src/test/java/org/opensearch/ad/settings/AnomalyDetectorSettingsTests.java

```diff
@@ -333,8 +342,10 @@ public void testSettingsGetValueWithLegacyFallback() {
             .put("opendistro.anomaly_detection.ad_result_history_max_docs", 8L)
             .put("opendistro.anomaly_detection.ad_result_history_retention_period", "9d")
             .put("opendistro.anomaly_detection.max_retry_for_unresponsive_node", 10)
+            .put("plugins.timeseries.max_retry_for_unresponsive_node", 10)
             .put("opendistro.anomaly_detection.cooldown_minutes", "11m")
             .put("opendistro.anomaly_detection.backoff_minutes", "12m")
+            .put("plugins.timeseries.backoff_minutes", "12m")
```

Comment:
question: are these backoffs surfaced to the user? Can be helpful to them in understanding delays etc.

Reply:
yes, they are surfaced to the user and user can adjust them.